conf/config.py

```diff
@@ -47,6 +47,9 @@ class BaseConfiguration:
     # Queue name to receive message from. For example:
     # Consumer.client-mts.queue.VirtualTopic.eng.mbs.module.state.change
     rhmsg_queue = 'Consumer.client-mts.queue.VirtualTopic.eng.mbs.module.state.change'
+    # The name used to identify unique subscriptions. Set this to a unique value
+    # to enable durable messages.
+    rhmsg_subscription_name = ''
```

Comment:
Why not default this to `None`? Then you can just pass it in `consumer.consume` every time.

Reply:
Just trying to follow the existing convention in this file.

Reply:
I think it makes sense to set None. Because, `rhmsg_subscription_name` is optional and rhmsg works with value `None`. The `rhmsg_certificate`, `rhmsg_private_key` and `rhmsg_ca_cert` are required, the empty string just a placeholder.

Reply:
Sure thing. Changed.